Sources/Customization/DefaultEventDispatcher.swift

```diff
@@ -38,22 +36,15 @@ open class DefaultEventDispatcher: BackgroundingCallbacks, OPTEventDispatcher {
         static public let maxQueueSize = 10000
         static let maxFailureCount = 3
     }
-    
-    // the max failure count.  there is no backoff timer.
-    
+        
     lazy var logger = OPTLoggerFactory.getLogger()
-    var backingStore: DataStoreType
-    var backingStoreName: String
-    
     // for dispatching events
-    let dispatcher = DispatchQueue(label: "DefaultEventDispatcherQueue")
+    let lock = DispatchQueue(label: "DefaultEventDispatcherQueue")
     // using a datastore queue with a backing file
-    let dataStore: DataStoreQueueStackImpl<EventForDispatch>
+    let queue: DataStoreQueueStackImpl<EventForDispatch>
```

Comment:
can we name it to datastoreQueue?

Reply:
I'll change it to "eventQueue" which makes more sense.